src/v1/error.js

```diff
@@ -44,6 +44,12 @@ class Neo4jError extends Error {
     super( message );
     this.message = message;
     this.code = code;
+    Object.defineProperty(this, 'name', {
```

Comment:
There was some reading on property definition vs assignment and by defining the property we are making it non enumerable.
http://2ality.com/2012/08/property-definition-assignment.html